src/app/Library/CrudPanel/Traits/Tabs.php

```diff
@@ -105,57 +105,65 @@ public function isLastTab($label)
         return $this->getLastTab() == $label;
     }
 
-    /**
-     * @return \Illuminate\Support\Collection
-     */
-    public function getFieldsWithoutATab()
+    public function getFieldsWithoutATab(): \Illuminate\Support\Collection
+    {
+        return $this->getElementsWithoutATab($this->getCurrentFields());
+    }
+
+    public function getColumnsWithoutATab(): \Illuminate\Support\Collection
```

Comment:
If we are creating a new function `getElementsWithoutATab()` I think we should not create another function for columns. 
Instead, we should deprecate the `getFieldsWithoutATab` with a notice that it will be removed in future versions and shouldn't be used.
Change where we use `getFieldsWithoutATab` to `getElementsWithoutATab` and in columns also use `getElementsWithoutATab`. 

Making it call the new function is fine but can't change the return type, (sorry 😞 ), it's a BC.